src/main/java/komo/fraczek/toukparking/resource/OperatorController.java

```diff
@@ -0,0 +1,41 @@
+package komo.fraczek.toukparking.resource;
+
+
+import komo.fraczek.toukparking.domain.ParkingBill;
+import komo.fraczek.toukparking.service.MeterRepository;
+import komo.fraczek.toukparking.service.ParkingService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@RestController
+public class OperatorController {
+
+    private static final Logger logger = LoggerFactory.getLogger(OperatorController.class);
+
+    @Autowired
+    ParkingService parkingService;
```

Comment:
minus za wstrzykiwanie polem

Reply:
Minus za brak konsekwencji również. Raz polem, raz setterem. Wydaje się, że nie wiesz jak to robić, więc robisz to byle jak. Tymczasem jest kilka sposobów, każdy jest inny i każdy ma inne wady/zalety, których trzeba być świadomy.